pkg/cloud/affinity_groups.go

```diff
@@ -21,17 +21,14 @@ import (
 	"github.com/pkg/errors"
 )
 
-const (
```

Comment:
Please revert this too. Mark intentionally made these consts. 

I don't particularly care whether there are string literals in the tests or not, but these changes will only aggravate the rebase I will have to do to brink in my dummies module.